refinery/ui/source/js/user-file-browser/ctrls/user-file-browser-files-ctrl.js

```diff
@@ -96,38 +98,23 @@
     };
 
     gridOptionsService.appScopeProvider = vm;
-    vm.downloadCsvQuery = function () {
+    function downloadCsv () {
+      document.location.href = '/files_download?' + downloadCsvQuery();
+    }
+
+    function downloadCsvQuery () {
       return $httpParamSerializer({
-        fq: createFacetQuery(),
+        filter_attribute: userFileFiltersService,
```

Comment:
missing method?

Reply:
No, the way Chuck was creating the facet query was redundant. I was able to reuse `filter_attribute` and the userFileFiltersService and remove `createFacetQuery`